CODEOWNERS

```diff
@@ -47,20 +47,37 @@
 /enos/     @hashicorp/quality-team
 
 # Cryptosec
-/builtin/logical/pki/         @hashicorp/vault-crypto
-/builtin/credential/cert/     @hashicorp/vault-crypto
-/builtin/logical/ssh/         @hashicorp/vault-crypto
-/builtin/logical/transit/     @hashicorp/vault-crypto
-/helper/random/               @hashicorp/vault-crypto
-/sdk/helper/certutil/         @hashicorp/vault-crypto
-/sdk/helper/cryptoutil/       @hashicorp/vault-crypto
-/sdk/helper/kdf/              @hashicorp/vault-crypto
-/sdk/helper/keysutil/         @hashicorp/vault-crypto
-/sdk/helper/ocsp/             @hashicorp/vault-crypto
-/sdk/helper/salt/             @hashicorp/vault-crypto
-/sdk/helper/tlsutil/          @hashicorp/vault-crypto
-/shamir/                      @hashicorp/vault-crypto
-/vault/barrier*               @hashicorp/vault-crypto
-/vault/managed_key*           @hashicorp/vault-crypto
-/vault/seal*                  @hashicorp/vault-crypto
-/vault/seal/                  @hashicorp/vault-crypto
+/builtin/logical/pki/                                @hashicorp/vault-crypto
+/website/content/docs/secrets/pki/                   @hashicorp/vault-crypto
```

Comment:
In case someone other than Steve is wondering about the order I added stuff in, I mostly searched for existing code (`/builtin/logical/pki`) and found the relevant section in docs and added them below.